src/pybind/ceph_argparse.py

```diff
@@ -327,7 +327,7 @@ class CephString(CephArgtype):
     """
     String; pretty generic.  goodchars is a RE char class of valid chars
     """
-    def __init__(self, goodchars=''):
+    def __init__(self, goodchars='', allowempty=True):
```

Comment:
is it intentional to keep default value to true?

Reply:
yes, defaulting to true is intentional